web/pdf_viewer.js

```diff
@@ -247,6 +251,9 @@ class PDFViewer {
       );
     }
     this.container = options.container;
+    this.#updateContainerHeightCss(this.container.clientHeight);
+    this.#resizeObserver.observe(this.container);
+
```

Comment:
Also, potentially just add this code in the "old place" instead since we're validating that `this.container` is actually a valid element in the `if`-block below:
https://github.com/mozilla/pdf.js/pull/15830/files#diff-4238fc7e96d72f4095859850f3d24a9298ec2dc8bd96437214a02dccbd48385cL333